### PR TITLE
Update listOfProjects.js

### PR DESCRIPTION
**Pull request by lemostrash**
- Interactive content in Go language

### CODE REVIEW

File: src/components/ProjectList/listOfProjects.js [hunk 1]
```diff
@@ -1167,5 +1167,13 @@ const projectList = [
     description: 'Osu! is a free-to-play rhythm game primarily developed by the community.',
     tags: ['Video Game', 'C#' , 'Music']
   }
```
Comment (Roshanjossey) [inline src/components/ProjectList/listOfProjects.js:1169]:
```suggestion
  },
```

**Review by Roshanjossey — CHANGES_REQUESTED** [review]
Just one syntax correction.

All the best for this project. I'm a big fan of exercism and learning language through TDD. I hope this project works well.

**Comment by Roshanjossey** [comment]
Thank you for making this change 🖖 ✨

**Comment by lemostrash** [comment]
Thanks for the opportunity.

**Pull request merged**